components/cli/pkg/minikube/init.go

```diff
@@ -67,7 +77,10 @@ func SetProfile(profile string) func(*Minikube) {
 	}
 }
 
-func ClusterExists(profile string) (bool, error) {
+func ClusterStatus(profile string) (Status, error) {
+	if !util.IsCommandAvailable("minikube") {
```

Comment:
shall we extract a constant for minikube? its used in multiple places.

Reply:
Updated.